File: challenge62.py
import sys

def mod(n, m):
    """Calculate the modulus without using the % operator.

    n / m returns the closest multiple, so if we multiply this by m and subtract
    from n we should have the modulus.

    """
    return n - ((n / m) * m)

test_cases = open(sys.argv[1], 'r')

for test in test_cases:
    n, m = test.strip().split(',')
    print mod(int(n), int(m))

test_cases.close()

File: Ruby/challenge62.rb
def mod(n, m)
  # Calculates the modulus without using the % operator
  return n - ((n / m) * m)
end

File.open(ARGV[0]).each_line do |line|
  n, m = line.strip().split(',')
  puts mod(n.to_i, m.to_i)
end
